ndmg/scripts/ndmg_dwi_pipeline.py

```diff
@@ -35,97 +36,178 @@
 import numpy as np
 import nibabel as nb
 import os
+from ndmg.graph import biggraph as ndbg
+import traceback
+from ndmg.utils.bids_utils import name_resource
+import sys
+
+
+os.environ["MPLCONFIGDIR"] = "/tmp/"
 
 
-def ndmg_dwi_pipeline(dwi, bvals, bvecs, mprage, atlas, mask, labels, outdir,
-                  clean=False, fmt='edgelist'):
+def ndmg_dwi_worker(dwi, bvals, bvecs, t1w, atlas, mask, labels, outdir,
+                    clean=False, fmt='csv', big=False):
```

Comment:
I'm quite confused about the format.... In two places the option for formatting of the graphs has been removed and set to different formats in each case, and now this still has a settable format, with the default value again being a third different format.

Reply:
Sorry, I mentioned above this is in 1 other place, but I see 3 un-addressed comments on this point @ebridge2 .

Reply:
removed format entirely as an option; dwi does 'elist'; fmri does 'adj'.

Reply:
That is not the desired behaviour - we said we would support both formats. It's inconvenient for graphs to be in two different formats if you're trying to work with both together. It should not be a difficult option to support.

Reply:
For the parser: 
```
parser.add_argument("--format", action="store", choices=["edgelist", "adjacency"], default="edgelist")
```

If `edgelist` is selected, the outputs will be `_elist.csv`, if `adjacency` is selected, the outputs will be `_adj.csv`.

This should be supported completely equivalently for both functional and diffusion.